src/types.ts

```diff
@@ -97,6 +97,7 @@ export interface AdvancedURISettings {
     useUID: boolean;
     addFilepathWhenUsingUID: boolean;
     allowEval: boolean;
+    includeVaultName: boolean;
```

Comment:
Isn't `vaultParam: 'id' | 'name'`  missing here? Should ts report an issue that `vaultParam` doesn't exist?

Reply:
Yes that's true. I will update `types.ts`.

There is a guard check at https://github.com/luckman212/obsidian-advanced-uri/blob/72597bd9e43f90c52d8737528da22b3288351bd6/src/tools.ts#L82 so it will fall back gracefully to using Vault name in case ID is null for some reason.